CONTRIBUTING.md

```diff
@@ -319,19 +319,23 @@ root@9feae5e5e001:/src/conjur-server#
 - A Google Cloud Platform account. To create an account see https://cloud.google.com/.
 - Google Cloud SDK installed. For information on how to install see https://cloud.google.com/sdk/docs
 - Access to a running Google Compute Engine instance. 
+- Access to predefined Google cloud function with the following [code](ci/authn-gcp/function/main.py).
 
 To run the cukes with a Google Cloud Platform (GCP) compatible environment, run `cli`
-with the `--authn-gcp` flag and pass a name of a running Google Compute Engine (GCE) instance:
+with the `--authn-gcp` flag and pass the following:
+1. The name of a running Google Compute Engine (GCE) instance
+
+2. The URL of the Google cloud function
```

Comment:
Wouldn't it be helpful to give an example?

Reply:
Added, please review again

Reply:
Google Cloud function 

Google Cloud is the name of the platform, so Cloud needs capital C